pkg/controller/nexus/update/monitor.go

```diff
@@ -0,0 +1,177 @@
+//     Copyright 2020 Nexus Operator and/or its authors
+//
+//     This file is part of Nexus Operator.
+//
+//     Nexus Operator is free software: you can redistribute it and/or modify
+//     it under the terms of the GNU General Public License as published by
```

Comment:
The headers must be updated to Apache.

Reply:
Ah, I thought `make test` would have done it. I guess I lost our bet then? Just switching the flag passed to `addlicense` did not replace existing headers. Bummer.

Reply:
No, that's why I had to use regexp to remove all headers first. :(